simulator/func_sim/operation.h

```diff
@@ -66,15 +67,19 @@ class Operation
 	//target is known at ID stage and always taken
 	bool is_direct_jump() const { return operation == OUT_J_JUMP; }
 
+	//target is known at ID stage; likely to be taken
+	bool is_likely_branch() const { return operation == OUT_BRANCH_LIKELY; }
```

Comment:
Please add tests for that condition to unit tests: https://github.com/MIPT-ILab/mipt-mips/blob/master/simulator/mips/t/mips32_branch_test.cpp

All common branches should have ```CHECK_FALSE( instr.is_likely_branch())``` and likely branches should have ```CHECK(instr.is_likely_branch())```